src/components/atoms/UserProfileInfo/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import styles from './styles';
+
+export default function UserProfileInfo({publications, likes, saved}) {
+  return (
+    <View style={styles.container}>
+      <View style={styles.space}>
+        <Text style={styles.info}>{publications}</Text>
+        <Text style={styles.title}>Publicaciones</Text>
+      </View>
+      <View style={styles.space}>
+        <Text style={styles.info}>{likes}</Text>
+        <Text style={styles.title}>Me gusta</Text>
+      </View>
+      <View>
+        <Text style={styles.info}>{saved}</Text>
+        <Text style={styles.title}>Guardados</Text>
+      </View>
```

Comment:
What do you think about creating a single component with the structure (Example):

```
<Space info={publications} title={'Publicaciones'} />
<Space info={likes} title={'Me gusta'} />
<Space info={saved} title={'Guardados'} />
```

And that it be called with the parameters that you are indicating?


```
     <View>
        <Text style={styles.info}>{info}</Text>
        <Text style={styles.title}>{title}</Text>
      </View>
```